iguana.corecontroller/src/main/java/org/aksw/iguana/cc/lang/impl/SPARQLLanguageProcessor.java

```diff
@@ -124,9 +123,9 @@ public static String getContentTypeVal(Header header) {
 
     public static long getJsonResultSize(String res) throws ParseException {
```

Comment:
This will become terrible to maintain. 
Also just to get the result size. Maybe we should remove the result size feature. 
This is too much hassle for too less output.

Reply:
Esp as this is only for SPARQL, not even generic